flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -106,6 +137,10 @@ class DataStreamOverAggregate(
         if (overWindow.lowerBound.isUnbounded &&
           overWindow.upperBound.isCurrentRow) {
           createUnboundedAndCurrentRowProcessingTimeOverWindow(inputDS)
+        } else if (overWindow.lowerBound.getOffset.getType.isInstanceOf[IntervalSqlType]
```

Comment:
@fhueske 
 As a personal preference I like to have the conditions as explicit as possible just to make sure it will not trigger in undesirable cases. I did not knew about the  isRows() function...of course I can check that.
Regarding the upper bound my reasoning was that if we will support other constructs in the future -e.g. FOLLOWING, fixed time boundaries...whatever else will be invented... it is much easier to integrate those if we have this explicit check now..rather then searching for it at that moment
However, if you prefer it removed of course it is not hard to do that :)